jsapp/js/components/activity/activityMessage.component.tsx

```diff
@@ -17,8 +17,11 @@ export function ActivityMessage(props: {data: ActivityLogsItem}) {
     .replace('##username##', `<strong>${props.data.username}</strong>`)
     .replace('##action##', props.data.action);
   // We only replace it if metadata is provided.
```

Comment:
I wonder if we should address somehow a (possible? is it even possible?) case when we have to display string with `##username2##, but the metadata is not there. Maybe this comment should also have something like 'If metadata is missing, we purposefuly display raw "##username2##" to make it obvious there's an error'. WDYT?